controller/storage/storage.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	model "github.com/kubernetes-incubator/service-catalog/model/service_controller"
+)
+
+// BindingDirection is an integer type used for expressing direction of the
+// service binding (from, to, or both).
+type BindingDirection int
+
```

Comment:
Hint of the future: there will be API client interfaces that look similar to these.